.buildkite/model_configs/default_config.yml

```diff
@@ -0,0 +1,295 @@
+test_edmf_consistency:
```

Comment:
So, this is replacing ArgParse?

Reply:
Can we put this one in a separate folder for easier findability?

Reply:
Yes, the goal to (almost) totally get rid of ArgParse, the only remaining entry being the `--config_file` argument.